backend/lib/endpoints/posts.js

```diff
@@ -233,6 +233,9 @@ async function routes(app) {
             title: true,
             types: true,
             visibility: true,
+            createdAt: true,
+            elapsedTimeText: true,
```

Comment:
I don't think that `elapsedTimeText` is part of the Post model, so you can remove it from the projection